### ISSUE TITLE
Test calling Rgemini from Python

### ISSUE THREAD

**Issue by vaakesan-SMH** [GEMINI-Medicine/Rgemini#32]
A python version of `Rgemini` will not be maintained in parallel with the R package, so for niche use-cases let's do some exploration of calling this package from Python using the `rpy2` package [here](https://github.com/rpy2/rpy2/tree/master).

Based on the findings, it would be useful to write up a discussion board post about this.

**Comment by julio-smh** [comment]
@loffleraSMH we could try running the testhat tests using rpy2, parse the output and then flag any fails, that way we would re-use the existing tests and could assure some level of compatibility.

**Comment by loffleraSMH** [comment]
> [@loffleraSMH](https://github.com/loffleraSMH) we could try running the testhat tests using rpy2, parse the output and then flag any fails, that way we would re-use the existing tests and could assure some level of compatibility.

@julio-smh: Yes, that sounds perfect - let's do that. Unfortunately, there are quite a few functions that currently don't have any unit tests yet (e.g., all functions requiring a DB connection), but our goal is to create a dummy DB allowing us to add unit tests for those functions in the future, and then we can add them to the rpy2 test.

If you have capacity to run the current tests using rpy2, feel free to assign yourself to this issue.
